### PR TITLE
Added community section to domestic landing page

### PR DESCRIPTION
**Pull request by erdem**

### CODE REVIEW

File: article/templates/article/sections/community_section.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="grid-row">
+      <div class="column-half-l">
+        {% progressive_image webp='images/export-hp-community.webp' img_id='business-is-great-image' src='images/export-hp-communtiy.jpg' %}
+      </div>
+      <div class="column-full column-half-l">
+        <h2 id="business-is-great-title" class="heading-large">Join our export community</h2>
+        <p id="business-is-great-description">
+          Our export community is a network of businesses who advocate for exporting. They share success stories, offer practical advice and lead by example.
+        </p>
+		<br>
```
Comment (eadpearce) [inline article/templates/article/sections/community_section.html]: `br` tags are unnecessary here as `p`s force a line break by default and the extra spacing between paragraphs here is inconsistent with the rest of the page

Comment (erdem) [reply article/templates/article/sections/community_section.html]: I didn't want to put that `<br>` tags as well but it used like this in all other section templates. The `br` tags adding an extra margin between elements. That's the reason being people used it before. 

Besides, updating syles for `p` tags can be raised consistency problem on FE side. If we are cool with this, I am happy to clean `br` tags.

Comment (eadpearce) [reply article/templates/article/sections/community_section.html]: No what I'm saying is we don't want the extra spacing here on the homepage so remove `br` tags. No need to update `p` styling

Comment (erdem) [reply article/templates/article/sections/community_section.html]: Actually, the new design has extra spacing in that section. I spoke with the rest of the team, no needs extra spacing anymore. I removed the tags.

**Pull request merged**